### PR TITLE
Fixed #110 and #111 in plotsignal

### LINKED ISSUE #110: plotsignal stepsize should be specified in seconds
it is now in ms, which is inconsistent with all other options in the ini files

### PR DESCRIPTION
**Pull request by stephenwhitmarsh**

### CODE REVIEW

**Comment by stephenwhitmarsh** [comment]
I forgot to check for changes in eegsynth master. Now configured upstream, and will pull from there before branching. Getting the hang of it :-)

File: module/plotsignal/plotsignal.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def butter_lowpass_filter(data, lowcut, fs, order=9):
 window    = int(round(window*hdr_input.fSample))
 clipsize  = config.getfloat('arguments','clipsize')
 clipsize  = int(round(clipsize*hdr_input.fSample))
-stepsize  = config.getfloat('arguments','stepsize')
+stepsize  = int(config.getfloat('arguments','stepsize') * 1000.0)
```
Comment (robertoostenveld) [inline module/plotsignal/plotsignal.py]: this does not have to be an int, a float is also fine.

**Comment by stephenwhitmarsh** [comment]
Are you sure? did you check?

**Comment by robertoostenveld** [comment]
Conceptually it does not have to be an integer, since time and sampling frequency are continuous variables. 

E.g. if I have an EEG system that samples at 1009 Hz (which happens to be a prime number), then I only get a sample that matches an integer number of milliseconds about every second (to be precise: every 1009 samples).

And if my sampling frequency is 100 times pi, i.e. 314.15926.... Hz, which happens to be an irrational number, then the only sample that is nicely aligned with an integer number of milliseconds is the very first (i.e. the sample at which t=0).

**Comment by robertoostenveld** [comment]
the code further down should be fixed so that it is not in milliseconds, but in seconds. And then it is obvious that it should be a float rather than an int.

**Pull request merged**